src/NServiceBus.StructureMap.Tests/NServiceBus.StructureMap.Tests.csproj

```diff
@@ -102,6 +102,8 @@
       <Name>NServiceBus.StructureMap</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
```

Comment:
This will constantly get added back in by everyone using the VS test runner. Why not simply include it?